### PR TITLE
Style summary

### PR DESCRIPTION
**Pull request by MiZiet**
hi, this will change look of page summaries in app according to the ui grant design.  

updated images:
image
image
image
image
image
image

### CODE REVIEW

File: packages/react-components/src/CardSummary.tsx [hunk 1]
```diff
@@ -106,21 +106,34 @@ export default React.memo(styled(CardSummary)`
...
     position: relative;
-    line-height: 1;
     text-align: right;
 
+    .ui--Labelled-content {
+      font-size: 1rem;
+      line-height: 1.214rem;
+      font-weight: var(--font-weight-normal);
+
+
```
Comment (krzysztof-jelski) [inline packages/react-components/src/CardSummary.tsx:118]: Remove empty line.

Comment (MiZiet) [reply packages/react-components/src/CardSummary.tsx]: ✅

File: packages/react-components/src/SummaryBox.tsx [hunk 2]
```diff
@@ -22,14 +22,19 @@ export default React.memo(styled(SummaryBox)`
   align-items: stretch;
   border-radius: 0.25rem;
   display: flex;
-  flex-wrap: no-wrap;
-  justify-content: space-between;
+  flex-wrap: nowrap;
```
Comment (jacogr) [inline packages/react-components/src/SummaryBox.tsx:25]: This is the first thing that caught my eye - certainly not a direction I believe we should go in, it looks all squashed into the left side now.

File: packages/react-components/src/styles/index.ts [hunk 3]
```diff
@@ -245,6 +255,22 @@ export default createGlobalStyle<Props & ThemeProps>(({ theme, uiHighlight }: Pr
     }
   }
 
+  .ui--SummaryBox {
+    > section {
+      background: ${hexToRGB(getHighlight(uiHighlight), '0.08')};
```
Comment (jacogr) [inline packages/react-components/src/styles/index.ts:260]: Why the background?

So at this point it actually catches your eye, why the summary is meant to be background info, not dominate.

File: packages/react-components/src/SummaryBox.tsx (hunk 2 above)
Comment (MiZiet) [reply packages/react-components/src/SummaryBox.tsx:25]: I've restored the `space-betweend` ✅

File: packages/react-components/src/styles/index.ts (hunk 3 above)
Comment (MiZiet) [reply packages/react-components/src/styles/index.ts]: I've removed the summary background ✅

**Comment by MiZiet** [comment]
hi, @jacogr could you take another look here?

**Comment by MiZiet** [comment]
hi @jacogr , updated with master and ready for review :)

**Comment by jacogr** [comment]
Closing. (https://github.com/polkadot-js/apps/pull/8725 probably brings it much closer to this)

**Pull request closed without merging**